test/CORS.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const test = require('ava')
+const listen = require('test-listen')
+const fetch = require('node-fetch')
+const mockedEnv = require('mocked-env')
+
+test('return no cors headers if env var specifies none', async (t) => {
+
+	const restore = mockedEnv({
+		ACCESS_CONTROL_ALLOW_ORIGIN: undefined
+	})
+
+	const server = require('../src/server')
```

Comment:
Why not adding this test to the server tests?

Reply:
For this test to work, we need to spin up a new server with a mocked environment, if I put this in `test/server.js`, I will need to refactor the module to create the server inside of each test. Thought this would be an easier path.